src/devices/CharacterLcd/CharacterLcd.csproj

```diff
@@ -9,17 +9,19 @@
 
   <ItemGroup>
     <Compile Include="Flags.cs" />
-    <Compile Include="GpioControllerAdapter.cs" />
-    <Compile Include="IGpioController.cs" />
     <Compile Include="Hd44780.cs" />
     <Compile Include="Lcd1602.cs" />
     <Compile Include="Lcd2004.cs" />
-    <Compile Include="Mcp23008Adapter.cs" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)../Mcp23xxx/Mcp23xxx.csproj" />
+    <!--
+      Temporarily making a project reference until the daily has IGpioController
+      <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" /> 
+    -->
+    <ProjectReference Include="..\..\System.Device.Gpio\System.Device.Gpio.csproj" >
+      <AdditionalProperties>Configuration=Linux-Debug;RuntimeIdentifier=linux</AdditionalProperties>
```

Comment:
NIT: You don't need to set Configuration, simply adding the RuntimeIdentifier should be sufficient.

Reply:
I was having problems in VS without it.